simple_config

```diff
@@ -12,5 +12,10 @@ path /echo EchoRequestHandler {}
 
 path /status StatusHandler {}
 
+path /proxy ProxyRequestHandler {
```

Comment:
On a similar note to the above, could we name this `/reverse_proxy ReverseProxyRequestHandler`?